keyboards/jisplit89/keymaps/default/keymap.c

```diff
@@ -0,0 +1,83 @@
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
```

Comment:
This file shouldn't be needed.  All the information that you'd need from this has functions for getting that info. 

```suggestion
```